docs/start/getting-started/fragments/flutter/add-api.md

```diff
@@ -2,15 +2,48 @@
 
 We will now use the Amplify CLI to configure the AWS Cloud Resources that will power your app. 
 
-1. Configure Amplify to manage cloud resources on your behalf. This step will configure a new AWS user in your account for Amplify. Open up a terminal window. You can use an external terminal or the integrated terminal in Android Studio. In the terminal, run:
+1. First ensure that you have installed the proper amplify cli version.  Within your terminal run: 
+
+    ```bash
+    amplify --version 
+    ```
+    Your output should be a version with "-flutter-preview" appended at the end. 
+
+    If not, run:
+
+    ```bash
+    npm install -g @aws-amplify/cli@flutter-preview
+    ```
+
+2. Initialize Amplify CLI by running: 
+
+    ```bash
+    amplify init
+    ```
+
+    Fill in the following values when running init: 
+
+    ```
+    ? Choose your default editor: Visual Studio Code
```

Comment:
This is conflicting to what @haverchuck just had in? Should this change?

Reply:
My bad - I suggested this revision using my personal preference

Reply:
good call I'll make it consistent